Public/Src/Pips/Dll/Operations/Process.cs

```diff
@@ -718,6 +723,16 @@ public sealed partial class Process : Pip
         /// </summary>
         public ReadOnlyArray<PathAtom> AllowedSurvivingChildProcessNames { get; }
 
+
+        /// <summary>
+        /// Process names that will break away from the sandbox when spawned by the main process
+        /// </summary>
+        /// <remarks>
+        /// The accesses of processes that break away from them sandbox won't be observed.
+        /// Processes that breakaway can survive the lifespan of the sandbox
+        /// </remarks>
+        public ReadOnlyArray<PathAtom> ChildProcessesToBreakawayFromSandbox { get; }

```

Comment:
>ChildProcessesToBreakawayFromSandbox [](start = 39, length = 36)

Should this be part of the fingerprint? I think it should because the pip will have a different behavior.

Reply:
Good point. Will add.

---
In reply to: [339171834](https://github.com/microsoft/BuildXL/pull/1099#discussion_r339171834) [](ancestors = 339171834)